docs/source/getting_started/quick_start.md

```diff
@@ -1,7 +1,7 @@
 # Quick Start
 
-Following is a simplified demo using example dataset to train a registration network.
-More details and other options can be found in the [command line tools](doc_command.md).
+This is a simple demo using an example dataset to train a registration network.
+More details and other options can be found in [Command line tools](cli.md).
```

Comment:
thx for fixing the wrong link, the `cli.md` might need to be replaced by `../docs/cli.md`

I didn't have the chance to add the docs for using readthedocs yet, but i belive you can test locally with `make clean html` under `docs/`